From d818b5456d64630ea018e5f70b5fa6f9f33b135e Mon Sep 17 00:00:00 2001
From: Andreas Baierl <ichgeh@imkreisrum.de>
Date: Fri, 3 Jan 2020 22:26:35 +0100
Subject: [PATCH 038/102] lima/parser: Fix rsw parser

Drop assert as it is not necessary and used wrong anyway.

Reviewed-by: Qiang Yu <yuq825@gmail.com>
Signed-off-by: Andreas Baierl <ichgeh@imkreisrum.de>
---
 src/gallium/drivers/lima/lima_parser.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/gallium/drivers/lima/lima_parser.c b/src/gallium/drivers/lima/lima_parser.c
index 2becad8c121..8850a284a0d 100644
--- a/src/gallium/drivers/lima/lima_parser.c
+++ b/src/gallium/drivers/lima/lima_parser.c
@@ -24,7 +24,6 @@
 
 #include "util/u_math.h"
 
-#include <assert.h>
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h>
@@ -522,7 +521,6 @@ parse_rsw(FILE *fp, uint32_t *value, int i, uint32_t *helper)
               *value & 0xffffffe0, *value & 0x0000001f);
       break;
    case 10: /* VARYING TYPES */
-      assert(*helper);
       fprintf(fp, "(1): ");
       int val, j;
       /* 0 - 5 */
-- 
2.24.1

